src/__tests__/utils.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-env node */
+/* eslint-disable header/header */
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
If you put comments in this order, they pass the linter without disabling

```suggestion
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
/* eslint-env node */
```